dashboard/src/components/Logo.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Logo = () => {
+  return (
+    <div className="row">
+      <div className="col s6 m6 l4 offset-s0 offset-m0 offset-l2 left-align">
+        <img className="header-logo" alt="Logo" src={require("../assets/seamless.me-logo.svg")} />
```

Comment:
instead of inlining require, import at the top of the file:
```
import React from 'react';
import seamlessLogo from '../assets/seamless.me-logo.svg';

const Logo = () => {
  return (
    <div className="row">
      <div className="col s6 m6 l4 offset-s0 offset-m0 offset-l2 left-align">
        <img className="header-logo" alt="Logo" src={seamlessLogo} />
```
same for header-logo